package.json

```diff
@@ -7,6 +7,12 @@
   "scripts": {
     "bench": "node benchmark",
     "clean": "lerna run clean",
+    "clean:examples": "find examples -name 'node_modules' -type d -prune -exec rm -rf '{}' +",
+    "clean:packages": "lerna clean -y",
+    "clean:examples:locks": "find examples -name 'yarn.lock' -type f -exec rm '{}' + && find examples -name 'package-lock.json' -type f -exec rm '{}' +",
```

Comment:
not compatible with windows

Reply:
also we are using npm now, you should not have yarn lock files

Reply:
ignore it, pushed accidently